common/app/Panes/redux/index.js

```diff
@@ -202,7 +201,7 @@ export default function createPanesAspects({ createPanesMap }) {
           return panesByName[name].isHidden ? sum : sum + 1;
         }, 0);
         let numOfHidden = 0;
-        return {
+        return numOfPanes < 1 ? state : {
```

Comment:
This is to prevent a single pane from being toggled off, when only one pane is being rendered, so the user never ends up with a blank screen.

Reply:
We have an issue and incoming PR for this will render a hidden message.